fireatlas/FireRun.py

```diff
@@ -11,6 +11,46 @@
 from fireatlas.utils import timed
 from fireatlas import settings
 
+def CreekFireforwardTestRun():
```

Comment:
I assume this is here b/c this was the run that produced the `tests/data/v2_outputs/*.fgb`?

Reply:
hmm, I guess if it is then shouldn't we call `convert_v2_pkl_to_csv` here? I see it mentioned in the `v3_run` test fixture

Reply:
Yeah so it gets a little convoluted due to the different (v2 vs. v3) environments involved. There may be a better way to organize this- I'm all ears if you have preferences.

Here is how the `tests/data/v2_outputs` were generated: I ran v2 using the exact settings on [this branch](https://github.com/Earth-Information-System/fireatlas/tree/files_for_unit_testing). Then, I manually ran `convert_v2_pkl_to_csv` _while in the v2 env_, so that I could open the pickles, and set the output path to `tests/data/FEDSpreprocessed`. Those are the files that I'm uploading here, and shouldn't ever need to be regenerated. 

`convert_v2_pkl_to_csv` is not called anywhere in the test, and wouldn't be needed unless someone is trying to generate a new test (say, for a different area). I wanted to put it under version control for reproducibility, though. 

Same story for `CreekFireforwardTestRun`. It is not actually used in the test, and I could take it out for sure. It is just defining the v3 equivalent of the job I ran with v2 for reproducibility purposes.

Reply:
Non issue. I was confused. I didn't see at first that the SNPP files came from `convert_v2_pkl_to_csv` b/c I was skimming too fast. I'm good with how it is. Those changes should stay 👍